docs/docs/how-to/add_users_to_project.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Adding users to your project
+sidebar_position: 100
+---
+
+## Prerequisites
+
+* The `minder` CLI application
+* A Stacklok account
+
+## Roles Overview
+When incorporating a user into your project, it's crucial to assign them the appropriate role based on their responsibilities and required access levels. 
+Minder currently offers the following roles:
+
+- `viewer`: Provides read-only access to the project. Users with this role can view associated resources such as enrolled repositories, rule types, and profiles.
+- `editor`: Grants the same permissions as the viewer role, along with the ability to edit project resources, excluding the project itself and the list of providers.
+- `admin`: Grants administrator rights on the project. Users with this role have the same permissions as editor and can also modify the project and associated providers.
+- `policy_writer`: Allows users to create rule types and profiles.
+- `permissions_manager`: Allows users to manage roles for other users within the project.
+
+## Adding a user
+To add a user to your project, follow these steps:
+
+1) Determine the User's Role: Decide the appropriate role for the user based on their responsibilities.
+
+2) Execute the Command:
+    ```bash
+    minder project role grant --sub user-id --role desired-role --project project-id
```

Comment:
how would I view the roles a user has on a project>

Reply:
Added it to the docs